source/features/index.tsx

```diff
@@ -51,6 +52,7 @@ const logError = (id: FeatureID, error: unknown): void => {
 	const message = error instanceof Error ? error.message : String(error);
 
 	if (message.includes('token')) {
+		void showToast(async () => Promise.reject(new Error(message)));
```

Comment:
Not necessary, but it'd be nice to use it as:

```suggestion
		void showToast(new Error(message));
```

and also the regular
```ts
		void showToast("My DONE notification");
```

in addition to the current situation